src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-form.tsx

```diff
@@ -24,9 +24,17 @@ import { useCallback, useEffect, useMemo } from 'react';
 import { useIntl } from 'react-intl';
 import { SHUNT_COMPENSATOR_TYPES } from '../../../../network/constants';
 import GridItem from '../../../commons/grid-item';
+import { ShuntCompensatorFormInfos } from '../shunt-compensator-dialog.type';
 
+export type CharacteristicsFormProps = {
+    previousValues?: ShuntCompensatorFormInfos;
+    isModification?: boolean;
```

Comment:
```suggestion
    isModification: boolean;
```
Why question mark if we have a default value ?